src/main/java/io/cryostat/net/web/http/api/v2/AbstractJwtConsumingHandler.java

```diff
@@ -164,8 +173,12 @@ private JWT validateJwt(RoutingContext ctx)
     protected ConnectionDescriptor getConnectionDescriptorFromJwt(RoutingContext ctx, JWT jwt)
             throws ParseException {
         String targetId = ctx.pathParam("targetId");
-        // TODO inject the CredentialsManager here to check for stored credentials
         Credentials credentials = null;
+        try {
+            credentials = credentialsManager.getCredentialsByTargetId(targetId);
+        } catch (ScriptException | IOException e) {
+            logger.error(e);
+        }
```

Comment:
Is this code from a rebase on top of https://github.com/cryostatio/cryostat/pull/1025 ?

Reply:
Yes, looks like it. I'll try to clean up the history.

Reply:
Done, I rebased the `939-story-pluggable-discovery-api` upstream branch so it's up-to-date with upstream main, and then rebase this feature branch of mine onto the `939-` branch. History should be clean now and the diff shouldn't contain any chunks from #1025 or other unrelated changes.